tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -138,6 +138,51 @@ steps:
                   exit 1
                 fi
             else
+              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+
+              if [[ $release == "release" && $feedId == "build" ]]; then
+                packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
+                
+                if [ "${{ parameters.artifactPath }}" == "scoped" ]; then
```

Comment:
It would be nice to describe why this special case in the comment or PR description.